python/guide/module/gluon/nn.md

```diff
@@ -245,12 +259,14 @@ chimera(x)
 
 ## Compilation
 
-The avid reader is probably starting to worry about the
-efficiency of this. After all, we have lots of dictionary lookups, code
-execution, and lots of other Pythonic things going on in what is supposed to be
-a high performance deep learning library. The problems of Python's [Global
-Interpreter Lock](https://wiki.python.org/moin/GlobalInterpreterLock) are well
-known. In the context of deep learning it means that we have a super fast GPU
+The reader may be starting to think about the efficiency of this Python code.
+After all, we have lots of dictionary lookups, code execution, and lots of
+other Pythonic things going on in what is supposed to be a high performance
+deep learning library. The problems of Python's [Global Interpreter
+Lock](https://wiki.python.org/moin/GlobalInterpreterLock) are well
+known. 
+
+In the context of deep learning it means that we have a super fast GPU
 (or multiple of them) which might have to wait until a puny single CPU core
```

Comment:
puny and awful are strange selection of words

Reply:
Yes they are. Done.